ginga/Control.py

```diff
@@ -576,8 +581,8 @@ def  _dl_indicator(count, blksize, totalsize):
             try:
                 dl = catalog.URLServer(self.logger, "downloader", "dl",
                                        info.url, "")
-                filepath = dl.retrieve(info.url, filepath=info.filepath,
-                                       cb_fn=_dl_indicator)
+                dl.retrieve(info.url, filepath=info.filepath,
+                            cb_fn=_dl_indicator)
```

Comment:
My `flake8` checker is complaining that `filepath` is defined by unused, that's why I deleted it. I don't see how it can be used in Line 649 (655 in this PR). See my next reply below.